### PR TITLE
Update chart dependencies

### PR DESCRIPTION
**Pull request by palexster**
# Description

This PR updates the Capsule chart used by Liqo to version v0.1.0. This prevents issues concerning v1beta1 CRDs (Ref. #842) and moves the dependency chart to the the official chart repository.

# How Has This Been Tested?

- [ ] E2E (Existing)

### CODE REVIEW

File: deployments/liqo/Chart.yaml [hunk 1]
```diff
@@ -23,7 +23,7 @@ version: "0.1"
 
 dependencies:
 - name: capsule
-  version: v0.1.0-rc2-liqo
+  version: v0.1.0
   # this is a temporary solution waiting for the new version of capsule to be released.
```
Comment (giorio94) [inline deployments/liqo/Chart.yaml:27]: Should be deleted

File: deployments/liqo/Chart.yaml [hunk 2]
```diff
@@ -23,7 +23,7 @@ version: "0.1"
 
 dependencies:
 - name: capsule
-  version: v0.1.0-rc2-liqo
+  version: v0.1.0
```
Comment (giorio94) [inline deployments/liqo/Chart.yaml:26]: The version should also be updated in the go.mod and in the makefile

**Review by giorio94 — COMMENTED** [review]
A bit unrelated from this, but the tenant api version should be probably updated from v1alpha1 to v1beta1.

**Review by aleoli — APPROVED** [review]

**Review by giorio94 — APPROVED** [review]

**Comment by isaacdan** [comment]
I still see this issue
Error: failed to install CRD crds/crd.yaml: unable to recognize "": no matches for kind "CustomResourceDefinition" in version "apiextensions.k8s.io/v1beta1"

name = "aad-pod-identity"

repository = "https://raw.githubusercontent.com/Azure/aad-pod-identity/master/charts"
chart = "aad-pod-identity"
version = "4.1.7"

**Pull request merged**